OpenSim/Common/Exception.cpp

```diff
@@ -67,6 +69,23 @@ exception()
 //#endif
 }
 
+Exception::Exception(const std::string& file,
+                     size_t line,
+                     const std::string& func) {
+    addMessage(file + ":" + std::to_string(line) + _msgSeparator + 
```

Comment:
This implementation forces the file/line/func stuff to be at the beginning of the message, which is probably not appropriate for OpenSim messages. Please revise so that these come at the end, or make it flexible enough for us to play with the formatting later.